sshdconfig/build.ps1

```diff
@@ -0,0 +1,223 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
You shouldn't need a custom `build.ps1` as the one in the root of the repo should build this